src/elements/about-block.html

```diff
@@ -18,6 +18,7 @@
 
             .container {
                 overflow: hidden;
+                margin: 0 auto;
```

Comment:
it already has this value https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/shared-styles.html#L119